protocol/0044-LIME-lp_mechanics.md

```diff
@@ -334,18 +339,40 @@ In the case of spot markets it will be transferred into the network treasury for
   `LP2` will receive a full `100` stake back whilst `LP1` will receive a total of `85` back into their general account with `15` transferred into the market's insurance account  (<a name="0044-LIME-026" href="#0044-LIME-026">0044-LIME-026</a>)
 - When LP is committed they are obliged to provide liquidity equal to their commitment size on both sides of the order book (<a name="0044-LIME-027" href="#0044-LIME-027">0044-LIME-027</a>)
 - For a market that is in opening auction and LP has committed liquidity:
-  - LP can increase their commitment and it will take affect immediately (<a name="0044-LIME-050" href="#0044-LIME-050">0044-LIME-050</a>) for spot (<a name="0044-LIME-054" href="#0044-LIME-054">0044-LIME-054</a>)
-  - LP can decrease their commitment and it will take affect immediately without incurring penalties (<a name="0044-LIME-051" href="#0044-LIME-051">0044-LIME-051</a>) for spot (<a name="0044-LIME-055" href="#0044-LIME-055">0044-LIME-055</a>)
-  - LP can cancel their commitment without incurring penalties (<a name="0044-LIME-053" href="#0044-LIME-053">0044-LIME-053</a>)
+  - LP can increase their commitment and it will take effect immediately (<a name="0044-LIME-050" href="#0044-LIME-050">0044-LIME-050</a>). For spot (<a name="0044-LIME-054" href="#0044-LIME-054">0044-LIME-054</a>)
```

Comment:
This is inaccurate: when a LP increases commitment then
1. it takes effect immediately for the purposes of LP stake supplied to the market (in particular we can leave a liquidity auction under the right conditions).
1. in terms of liquidity they're expected to supply: this only takes effect from the start of the next epoch.

Reply:
updated AC and added one more AC for Liquidity Auction scenario

> - Consider a market in liquidity auction, when a LP increases their commitment it will take effect immediate for the purposes of LP stake supplied to the market. Where LP `supplied stake > target stake` the market will leave liquidity auction when the liquidity auction ends
>   - In terms of the liquidity they are expected to supply: this only takes effect from the start of the next epoch
>   (<a name="0044-LIME-102" href="#0044-LIME-102">0044-LIME-102</a>)